### PR TITLE
Custom welcome screen

### PR DESCRIPTION
**Pull request by daileytj**
Fixes PXBLUE-1948.

#### Changes proposed in this Pull Request:
- Add example pre-auth screen

To test this uncomment lines 15, 16, and 50 in the example App.tsx file, then navigate to http://localhost:3000/pre-auth

### CODE REVIEW

**Comment by daileytj** [comment]
On second glance at the other projects, I realized that this doesn't automatically load up the pre-auth screen as a "welcome" screen. I'll need to add an initialRouteName prop to handle this. Converting this to a draft until I get that finished.

**Comment by joebochill** [comment]
> On second glance at the other projects, I realized that this doesn't automatically load up the pre-auth screen as a "welcome" screen. I'll need to add an initialRouteName prop to handle this. Converting this to a draft until I get that finished.

Not sure if you need that or not. That was needed on the mobile side because, unlike a URL, you can just put in the path you want to go to when you open the app.

You should be able to just load up `localhost:3000/pre-auth` and have it work as expected (I'd also like to check if you can set up the `/` route to be a unauthenticated route).

**Comment by daileytj** [comment]
> You should be able to just load up `localhost:3000/pre-auth` and have it work as expected (I'd also like to check if you can set up the `/` route to be a unauthenticated route).

Sounds good. It's working as expected and you can set up the '/' route to be an unauthenticated route, so this should be good to go.

**Review by joebochill — APPROVED** [review]
One thing you might want to spend a few minutes exploring is seeing if you can skip showing the 'Loading Spinner' unless the route you are trying to go to is protected (i.e. for a public route, just load the route without the spinner). Otherwise this looks good 👍 

That spinner is on line 77 of `AuthNavigationContainer`

File: login-workflow/src/screens/AuthNavigationContainer.tsx [hunk 1]
```diff
@@ -72,9 +72,14 @@ export const AuthNavigationContainer: React.FC<NavigationContainerComponentProps
     }, []);
 
     const { routes, routesArray: publicRoutes } = prefixRoutes({ ...defaultRoutes, ...routeConfig });
+    const extraRoutesPathnames = extraRoutes.map((route) => route.props.path);
 
     // Show the splash loading screen until we have loaded the initial authentication state
-    if (securityState.isLoading) {
+    if (
+        securityState.isLoading &&
+        !publicRoutes.includes(window.location.pathname) &&
```
Comment (joebochill) [inline login-workflow/src/screens/AuthNavigationContainer.tsx:80]: Looks good, I would just take out this line. We will want to show the loading spinner when you are accessing one of the login-workflow screens or a private route. Just not for public (extra) routes.
```suggestion
```

**Pull request merged**